frontend/tests/page-object/base.page.ts

```diff
@@ -0,0 +1,8 @@
+import type { Page } from "@playwright/test";
+
+export const BasePage = {
+    goto: async (page: Page, endpoint: string) => {
+        await page.goto("https://v2-staging-mainnet.beaconcha.in" + endpoint);
+        await page.waitForTimeout(2000);
+    },
```

Comment:
```suggestion
export const basePage = {
    goto: async (page: Page, path: string) => {
        await page.goto("https://v2-staging-mainnet.beaconcha.in" + path);
        await page.waitForTimeout(2000);
    },
```

🤔 don't know if we should `stick` to your `PascalcCase convention` for `classes` or go with the `default` in `js`

Reply:
I would rather use PascalCase, just because in tests it will give more visability that this is for the specific page like await BasePage.goto(page)